blocksuite/affine/all/src/effects.ts

```diff
@@ -31,4 +32,5 @@ export declare const _GLOBAL_:
   | typeof componentToolbarEffects
   | typeof componentToggleButtonEffects
   | typeof componentColorPickerEffects
-  | typeof componentViewDropdownMenuEffects;
+  | typeof componentViewDropdownMenuEffects
+  | typeof componentResourceEffects;
```

Comment:
if no errors reported in `yarn typecheck` without this line, you can remove this. These effects will be removed in the future.